packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -370,6 +370,7 @@ export class ProjectActions {
 
   async reconfigureProject () {
     await this.api.closeActiveProject()
+    await this.initializeActiveProject()
```

Comment:
Seems you can remove line 372 if you like - `initializeActiveProject` does `closeActiveProject` for you.

https://github.com/cypress-io/cypress/blob/10.0-release/packages/data-context/src/actions/ProjectActions.ts#L123